### DISCUSSION TITLE
关于中文插件开发与使用文档编写及维护

### DISCUSSION THREAD

**Discussion by windingwind** [zotero-chinese/wiki#89] [社区建设]
大家好，我设想编写维护一份中文的插件开发文档，同时考虑是否需要将我的几个插件的文档做一份中文版，也一并归到此团队？
另外，我目前在飞书平台上编写文档，看起来协作、编辑与评论比较方便。可能本仓库的一些文档也可以在这类协作工具上创建副本，用于分享？
希望听听大家的想法。

**Comment by northword** [comment]
> 我设想编写维护一份中文的插件开发文档，同时考虑是否需要将我的几个插件的文档做一份中文版，也一并归到此团队？

支持！

插件的中文文档，如果是面向用户的话，我觉得可以归入 Zotero 手册。我之前在 https://github.com/orgs/zotero-cn/teams/all/discussions/2/comments/7 这个讨论里提到想要维护一份单独的 Zotero 手册，就是想不仅包括 Zotero 本身的使用，也包括一些好用插件的使用。

如果是面向开发者，例如 api 等，可以放进开发文档。


> 另外，我目前在飞书平台上编写文档，看起来协作、编辑与评论比较方便。可能本仓库的一些文档也可以在这类协作工具上创建副本，用于分享？

还没有用过飞书，用过 GitBook ，感觉协作也不错，并且可以将源文件双向到 GitHub 仓库[^gitbook-sync]。
其实我最开始设想的只是用 Markdown 写，Mkdocs/VuePress 之类的工具构建网页，Actions 自动部署。不过对于有很多人同时写的话，确实不如实时协作工具方便。这个大家可以讨论一下用哪种方案。

~~UPDATE: 简单看了下 飞书/云文档/知识库 的帮助文档，感觉和语雀是差不多的，二者与 GitBook 的定位稍有不同，虽然三者都是都是知识管理，但 GitBook 可能更偏向对外的知识分享，例如书籍、文档等。
GitBook 的入门门槛可能要比前两者高一些，比如英文界面，需要了解 Git 和 PR 概念等，本土化可能也不如前二者。
另外， @l0o0 之前有提到注册域名，如果 GitBook 可以在国内访问的话，倒是可以把文档子域名绑定到 GitBook。如果我没看漏或看错的话，飞书和语雀（各自免费版）目前还不支持绑定域名。~~
算是各有优缺，可以作为一个选择。

再次 UPDATE：不好意思，看错了，GitBook 个人免费，创建组织是付费的 orz，不过开源项目似乎可以申请免费。

[^gitbook-sync]: https://docs.gitbook.com/integrations/git-sync
---

之前没有收到这条讨论的邮件通知，可能是需要 @zotero-cn/all 这样 call 一下大家。

**Reply by windingwind** [reply]
谢谢建议，域名的事情之前也和@l0o0老哥简单讨论过，能够在可承受的维护成本前提下保证稳定和快速访问其实还是很好的。但正如在另一个讨论中提到的，除非自建网站，否则像Github这样的平台在国内的访问稳定性是个很大的麻烦。但自建网站的维护成本又比较高。

相比域名，我提供一种基于社交媒介的思路：是否可以使用例如微博等平台创建一个账号，将一些文档、资源（例如CSL）放在账号上。社群内的分享只需要通过类似“微博搜索xxx”即可找到相关资料，而不需要依赖于固定域名。
好处如下：
- 维护成本小。如上所说，要将我们的社群资料（代码+文档）放到域名下，并保证国内用户不翻墙也可以使用，需要较高的维护成本（精力+费用）；一个社交媒体账号维护只需要定期登录即可。
- 适合新世代用户习惯。目前第一批00后学生已经步入研究生阶段，在接下来的数年中越来越多的用户将原生于移动时代，而非web时代。相比一个通过链接访问，新世代用户可能更习惯在社交媒体上关注并互动。
- 可以起到和域名相同的作用。域名的主要目的之一是保持社群联系，将链接放在社交媒体账号中，随时更新、替换内容也不会导致用户丢失对社群资料的访问。
- 社交媒体对推广的天然亲和性。尽管非盈利，社群维护需要推广来保证新鲜血液。社交媒体天然具有推广属性。

同时社交媒体和域名、github等平台也不冲突。可以一起使用。

当然这个意见尚不成熟，还需要更多的讨论。@zotero-cn/all

**Reply by windingwind** [reply]
我自己认为是需要一个：
- 以团队发布内容为主的
- 可以方便稳定访问的
- 支持长文档阅读的
平台。

思考了一下国内常用的一些社交平台：
- 微信
微信公众号自从改成流式推送以后经常容易丢内容，且评论功能受限，不太适合。
- QQ
群不方便搜索，群组消息太容易被刷掉，目前的qq群还是起到答疑作用，且人数受限，不是团队中心的环境，不太适合
- 贴吧
娱乐性比较强，不再像以前一样适合专业性的讨论
- 小红书
只有移动端，不适合长文，放弃
- B站
不适合长文，放弃
- 微博
似乎可以
- 豆瓣
豆瓣群组似乎可以

结论是可以参考微博或者豆瓣的平台，看是否可以将内容或内容链接放在上面，方便用户保持访问？

**Reply by zepinglee** [reply]
我不太赞成用社交平台。社交平台推送新闻之类的还可以，但是像提供 CSL 样式供用户下载就很不方便。

**Reply by northword** [reply]
社交媒体适合一些短、彼此相对独立不成体系、或有一定时效性的内容，文档之类的放在一个层次分明的地方维护比较好。

如果只发链接的话，放在社交媒体上的就成了xxx文档的一个链接，这其实和没有区别不大 -- 比起给别人说去微博搜xxx，我觉得访问链接可能更方便。

或者就是把文档拆分，发在社交媒体上，但这样一方面失去了层次感，变得零碎。另一方面，社交媒体的属性天然不适合长文，更适合零散的、碎片化的东西。
（很多人在社交媒体上不会去看长文和干货，看到了->收藏了＝我会了）
这一点其实可以一定程度参考青柠学术的公众号，基本上都是比较零碎的点，很适合刷社交媒体时候顺手点进去读一下。
在公众号上，相似主题的通过标签和合集进行区分，但仍然不如阅读文档。

所以我觉得，社交媒体或许可以有，更多是作为宣发渠道。文档/知识库 这类专门、集中的平台还是要有，不管是在 github 构建网页，还是飞书、语雀、gitbook之类的。

另外b站和知乎也可以写长文，我有些资料就在知乎找的 😂

**Reply by windingwind** [reply]
> 我不太赞成用社交平台。社交平台推送新闻之类的还可以，但是像提供 CSL 样式供用户下载就很不方便。

我同意供用户下载不够方便，我也同意社交平台不是最佳选择。
但是考虑到许多用户不知道/无法找到/无法访问不管是github还是gitee的资源及文档的现状，以及维护一个固定的域名并不那么方便的情况，我目前感觉社交平台是无奈之选。
CSL仓库的主页上有各种文档，包括各个平台上还有许多zotero使用的文档。目前zotero的中文社群里，我认为并不是文档太少，而是文档太多，没有统一的渠道与梳理。许多文档内容过时（最典型的是zotero6以后zotfile+网盘的同步方式已经事实上不再是首选，但许多旧文档中仍然在主推），许多文档的作者甚至都不一定熟悉zotero，只是使用了一两周就开始在网上做经验分享。长期以来社区的经验积累只能靠口口相传的现状已经成为了社区发展的重要阻碍。

我认为借这个中文团队建立的势头，如果能够组织起一个相对可靠的zotero社区意见中心（不论是域名或是社交媒体等其他形式），都会推动整个社区的巨大进步。否则，社区的现状仍然无法真正改变，目前由I0o0维护的zotero中文群（qq）中每天充斥着大量重复、基础的安装、同步、样式问题，就是这一现状的印证。

回到社交平台的问题，我对其的定义是一个稳定可靠方便分享的社群入口。zotero用户碰到问题可以首先“去微博zotero中文看看”，这样就解决了大量的麻烦。包括csl，也可以将其链接放在社交平台上，和用户下载并不矛盾，相反是有益于用户找到下载入口的。

**Reply by windingwind** [reply]
@northword 
完全赞同，我也是设想把社交媒体当作一个链接入口，用来替代域名，作为一种解决“到哪里找更新及时、正确、全面的zotero文档”的方案。
不需要把所有文档或材料放在社交媒体，只是在上面建一个索引。用户可以通过搜索功能快速找到。
具体选择哪个平台放文档，和使用域名还是社交媒体作为入口，我感觉是两个不同的问题。

**Reply by northword** [reply]
话说，其实可以注册域名，隐式转发到 gitee pages 不知道可不可行。
资源在github维护（action），镜像到gitee。反馈搜集可以社交媒体、gitee issue、群。

**Reply by windingwind** [reply]
域名隐式转发类似代理了，应该需要服务器吧？

**Reply by northword** [reply]
> 域名隐式转发类似代理了，应该需要服务器吧？

部分域名 dns 解析商是提供隐式转发的，阿里云就可以，不过免费版好像是限制转发两条（似乎也够了）

**Reply by northword** [reply]
就是不知道他是只能转发域名根页面还是说子页面也可以。
比如 zotero-chinese.cn转发到zotero-chinese.gitee.io，

访问
https://zotero-chinese.cn正常，在里面点个链接如 /styles，是不是能保持https://zotero-chinese.cn/styles，还是会变成zotero-chinese.gitee.io/styles

我回头试一下，如果可以的话或许这是个不错的方案，可以首页放各个板块的链接

**Reply by windingwind** [reply]
再啰嗦一点为什么我觉得需要一个域名/社交平台来指向一份由我们中文组维护的，更新及时、正确、全面的zotero文档（不管是基础的安装使用，还是插件，还是CSL）呢？

因为我在qq群里每天都看到同类的问题，我自己维护插件，也会反复看到一大堆别人问过的，在网上搜得到解决办法的问题。

那么为什么网上这么多帖子但是用户还是会反复问呢？一方面是一部分人不善于搜索资源，对于这部分人告诉他“去搜xxx”或者扔一个链接就可以；另一方面是有些人跟着网上错误的/过时的帖子操作，导致出现问题。

对于这些情况，在群里答疑的人今天丢一个知乎链接，明天丢一个公众号链接，一方面是很麻烦，一方面是没法形成用户解决问题的习惯路径。今天解决了这个问题，明天他碰到另一个问题还是会问，而且对于其他人的问题，他也没法帮助解答。

如果有一个域名/社交媒体，除了特别罕见的问题，直接扔给他们一个链接就行。下次碰到链接，他们还可以去这里找解答，也可以很方便地在用户之间传播。

**Reply by northword** [reply]
> 再啰嗦一点为什么我觉得需要一个域名/社交平台来指向一份由我们中文组维护的，更新及时、正确、全面的zotero文档（不管是基础的安装使用，还是插件，还是CSL）呢？

> 如果有一个域名/社交媒体，除了特别罕见的问题，直接扔给他们一个链接就行。下次碰到链接，他们还可以去这里找解答，也可以很方便地在用户之间传播。

同感！

**Reply by windingwind** [reply]
@northword 
能够直接对域名做转发的话就太好了。

**Reply by windingwind** [reply]
@northword 

飞书可对团队的域名进行修改（仅限一次），成为类似：zotero.feishu.cn 的形式。我认为也可以满足我们的需要

**Reply by northword** [reply]
> @northword 
> 
> 飞书可对团队的域名进行修改（仅限一次），成为类似：zotero.feishu.cn 的形式。我认为也可以满足我们的需要

这个方案无法和 styles 仓库对接起来，并且这样子与 zotero-chinese.gitee.io 是类似的。

可以作为域名隐式转发不可用的备选方案之一。

**Reply by windingwind** [reply]
同意可作为备选。另一种考虑是可以将style仓库等资源的入口放在某处。
gitee的编辑功能太弱且访问速度不如飞书，提供的文档格式也不够丰富，在这点上我投飞书/语雀等类似工具一票

**Reply by northword** [reply]
> 同意可作为备选。另一种考虑是可以将style仓库等资源的入口放在某处。

资源入口我是打算类似  https://zotero-cn.github.io 这种（临时的，不是最终版本），有一个整个项目的主页，提供组织维护的所有资源的链接。


> gitee的编辑功能太弱且访问速度不如飞书，提供的文档格式也不够丰富，在这点上我投飞书/语雀等类似工具一票

我是偏向使用各种文档工具，例如 mkdocs，vuepress 等的，这些工具提供的文档格式已经满足了文档需要。并且大家无须另外注册账号，都用 github 账号即可。编辑时候使用 markdown 书写，提交后 action 自动构建，部署，镜像。

这个可以讨论一下。

**Reply by windingwind** [reply]
github的访问恐怕还是会有点问题，在不做转发的情况下。
我考虑使用在线协作工具的出发点是，希望文档维护过程中可以尽量吸收社区内的热心用户加入，也可以更好地减轻我们的负担。对于许多用户来说，且不论访问github的困难，让他们学习使用这些需要配环境/涉及代码的工具，不太友好。不论我们作为熟悉代码的群体，如何觉得它们简单，对新手来说难度太大。
文档的维护涉及大量的工作，我认为不引入其他成员，只由我们几个人来做是不太现实的。

**Reply by northword** [reply]
我一直考虑优先用代码文档工具的考虑是可以在一个相对集中的地方，在同一域名（无论是自行注册的一级域名，还是github等分配的二级域名）下维护所有的内容，就像目前布局的：
- 主页：https://zotero-cn.github.io 
- zotero文档：https://zotero-cn.github.io/e-zotero-md
- 插件商店：https://zotero-cn.github.io/plugins （还没） 
- styles 选择：https://zotero-cn.github.io/styles 
- styles 文档：
https://zotero-cn.github.io/csl-doc-chinese 
所有的内容都是在一个域名下，看起来就是一个团队维护的。所有的维护工作也都在github/gitee，不用多个账号。

如果选用了其他平台且平台不支持绑定域名，就变成了多个地方，因为部分资源页面是自动构建的，无法迁移到飞书等平台。不过这一点如果做好各个资源之间的链接的话也可以接受。另外使用这些平台的话，数据能否安全的迁移出也是需要考虑的。


也确实如你所说，其他人编辑一方面面临访问问题，另一方面面临门槛高的问题。如果其余人只是简单编辑一两个页面，很少的内容的话，可以用页面里的编辑页面链接直接到 github/gitee 的文件编辑页面，提交后就可以，也不需要配置环境。但如果需要编辑多个页面或比较大量内容，这就比较麻烦了。
虽然我觉得可能有精力、能力且热心编写的人可能不太多，但有的话，也确实需要考虑一下这个问题。

**Reply by windingwind** [reply]
能在一处维护再好不过。目前我先整理我这边的文档和资源，后续等平台搭起来可以再合并

**Reply by northword** [reply]
我刚刚试了一下域名隐式转发，大致原理就是 DNS 服务商提供了一台服务器作 iframe 框架。
好消息是可以转发，坏消息是因为 iframe 的原因，url 路径是不会改变的。

例子：
转发到的网址：https://blog.northword.cn
被转发的网址（隐式）：https://zotero.northword.cn
（请忽略这个几百年没更新而且一点点含金量都没有的博客，已经没在用了 :( ）

在 https://blog.northword.cn 中访问某个页面，url 会变为 https://blog.northword.cn/post/iframe-autoHeight.html ，
而在 https://zotero.northword.cn 里，无论访问哪个页面，url 都只会是 https://zotero.northword.cn 。如果是访问设置了新页面打开的链接，就会打开到新页面，url 也改变。

做个主页放各种资源的链接倒是可以。

**Reply by l0o0** [reply]
> 我不太赞成用社交平台。社交平台推送新闻之类的还可以，但是像提供 CSL 样式供用户下载就很不方便。

社交平台可以是我们曝光的渠道，目前来看我在小红书上分享Zotero相关资料，反馈效果还是可以的，可以让更多同学知道我们



> 我刚刚试了一下域名隐式转发，大致原理就是 DNS 服务商提供了一台服务器作 iframe 框架。 好消息是可以转发，坏消息是因为 iframe 的原因，url 路径是不会改变的。
> 
> 例子： 转发到的网址：https://blog.northword.cn 被转发的网址（隐式）：https://zotero.northword.cn （请忽略这个几百年没更新而且一点点含金量都没有的博客，已经没在用了 :( ）
> 
> 在 https://blog.northword.cn 中访问某个页面，url 会变为 https://blog.northword.cn/post/iframe-autoHeight.html ， 而在 https://zotero.northword.cn 里，无论访问哪个页面，url 都只会是 https://zotero.northword.cn 。如果是访问设置了新页面打开的链接，就会打开到新页面，url 也改变。
> 
> 做个主页放各种资源的链接倒是可以。

你这个域名和后台服务器是自行购买的么？国内还是国外的？

**Reply by northword** [reply]
> 社交平台可以是我们曝光的渠道，目前来看我在小红书上分享Zotero相关资料，反馈效果还是可以的，可以让更多同学知道我们
> 

是的，这个我同意，作为曝光渠道，同时可以放一些比较零散点。


> 你这个域名和后台服务器是自行购买的么？国内还是国外的？

是的，都是国内的，现在都在腾讯云。服务器用的是腾讯轻量云，是到 12 月左右到期，但是网页内容目前基本上还是在 GitHub pages ，并没有迁移到服务器。
服务器目前对我来说最大的用处就是 frp 内网穿透学校的超算。下一次续费就太贵了（买时候 100/年，续费700+/年）所以不打算续费了。

**Reply by windingwind** [reply]
iframe方式的转发基本上宣告没啥大用了，用户上不去github的还是上不去。只能像northword所说，给团队主页设置一个好记的域名。

**Reply by l0o0** [reply]
@northword 那我准备测试一下，看看访问速度如何

**Reply by northword** [reply]
> iframe方式的转发基本上宣告没啥大用了，用户上不去github的还是上不去。只能像northword所说，给团队主页设置一个好记的域名。


> 那我准备测试一下，看看访问速度如何

要补充一下，转发到的地址必须是备案的，所以只能转发到 gitee ，不能转发到 github ，除非我们不在国内买域名/不用国内的 dns 服务商。

**Reply by windingwind** [reply]
转发到gitee也许还可以，折衷的办法也只有这个了

**Reply by northword** [reply]
大家好，

几天前在 Gitbook 申请了他们的公益计划，可以对于开源项目完全免费，目前已经通过了，我简单体验了一下，和飞书一样都很简洁，比起飞书的优势在于

- 可以绑定域名，
- 每个知识库，每个页面都可以自定义一个 slug，如果不自定义，就是转拼音。
- 可以给组织设置一个默认知识库，例如有多个知识库 zotero-guide， dev-guide 等，设置默认库为 zotero-guide，假设绑定域名 doc.zotero-chinese.com ，则访问 doc.zotero-chinese.com 时是自动跳转到 doc.zotero-chinese.com/zotero-guide 。对应飞书是一串自动生成的后缀，如果只记域名 w6zjinn49j.feishu.cn 会要求登录。
- 可以导出为 markdown ，并与 github 双向同步。

但劣势也很明显：
- 国内访问速度，我关掉梯子试了试，速度是有下降的，在测速网站测得结果，虽然都可以访问，但访问速度相差悬殊，快的不到 1 秒，慢的要多达 5 秒。
- 没有评论

另一个劣势是之前有提到过的需要了解一些 pr ，但是用过之后感觉这个还好，在网页端编辑，区别只是要点一下“edit”，编辑完要点一下“merge”，点 edit 时，gitbook 会自动创建一个 pr，保存时自动合并，倒是不需要命令。

gitbook 预览地址：https://zotero-chinese.gitbook.io/

也看了看语雀，本来看着语雀官方的页面 https://www.yuque.com/yuque 这种很不错，但是我愣是没找着怎么开启这个页面，并且语雀不如飞书简洁，所以放弃了。

也就是如果没有其他选项，我们可以在飞书和 gitbook 里选一个。如果我们确定要买域名，那可以考虑 gitbook，如果不买，那飞书已经满足了我们的需求。

目前文档主要在 GitHub 和 飞书 两个地方，大家可以决定一下，可能的话，可以把文档迁移到一个地方维护。

大家有什么想法嘛？

**Reply by windingwind** [reply]
看了下语雀，如果要团队的知识库“所有人可见”，需要付费or以非赢利组织形式申请。

**Comment by windingwind** [comment]
Better Notes插件的用户文档已经完成，目前放在飞书平台。
https://w6zjinn49j.feishu.cn/wiki/space/7132878100186087428?ccm_open_type=lark_wiki_spaceLink

**Comment by windingwind** [comment]
Zotero插件的开发者文档正在锐意组织中。飞书平台链接：
https://w6zjinn49j.feishu.cn/wiki/space/7133163790413185052?ccm_open_type=lark_wiki_spaceLink

目前完成了大致框架，还待后续补充内容

**Reply by windingwind** [reply]
@zotero-cn/all 
Zotero插件的开发者文档已经完成编写，感谢@l0o0 @iShareStuff 的文档编写工作！
后续可待cn组织的网站建立后合并到一起。

**Reply by windingwind** [reply]
插件开发文档增加bootstrap模式的相关内容。

**Comment by windingwind** [comment]
zotero插件框架模板已经公开在https://github.com/windingwind/zotero-addon-template

**Comment by windingwind** [comment]
另，下一阶段计划将现有的关于Zo使用的常见问题的最新的解答整理到飞书文档，方便搜索，与上述几个文档一起。飞书地址：https://w6zjinn49j.feishu.cn/wiki/space/7134644190071242780?ccm_open_type=lark_wiki_spaceLink

**Reply by northword** [reply]
早上好！我第一次用飞书，不太了解，昨天我申请了编辑权限，但是似乎申请编辑只能申请一页的，而不是知识库的，目前我只有“文档简介”这一页的编辑权限，有什么可以编辑多页的方法嘛？

**Reply by northword** [reply]
我参考 zotero 官方文档和韩老师的《优雅》，在“Zotero 使用教程”部分列了一个简单的大纲，大家看可以嘛？可以的话我们就可以填内容了🙌

**Reply by windingwind** [reply]
感谢北词，我在想是否需要区分“入门教程”和“详解教程”，入门教程类似于最小实现，用最少的操作把zotero用起来，适合不懂电脑的新用户；详解教程可以提供更多解释和设置的介绍，或者指向其他部分的链接，方便用户自定义。

**Reply by windingwind** [reply]
两个部分的具体形式不一定是各自单列，也可以是穿插或者用链接形式把详解的内容串到入门里面，目前还是一个简单想法，欢迎补充讨论。
出发点是线性的使用文档看起来很长一串，用户可能看了害怕或者没耐心走完

**Reply by northword** [reply]
是的，我也这么想。我设想中，“快速开始”这一页就是承担“入门教程”这个作用，类似于官方文档 https://www.zotero.org/support/quick_start_guide 这一页。

**Reply by windingwind** [reply]
快速开始或者可以单列一节，后续的内容可能归到类似“配置/设置”之类的一个大章节？
给用户一种“走完这节就可以开始用”的积极反馈。然后通过一些例如：要从知网下文献，请按照xxx（链接）操作 的方式将后面的内容挂到快速开始的末尾。
现在看起来快速开始和其他内容并列，里面还层层叠叠的，我看了都有点害怕xs

**Reply by windingwind** [reply]
BTW，刚才发现一个比较严重的问题，飞书文档的搜索功能只有登录后才有，之前一直没发现（因为一直登录着）。我在想开始正式工作之前，讨论下是不是应该换到语雀？我看语雀的官方文档是可以不登录搜索的。
我去试下注册一个语雀账号，能不能不登录搜索。

**Reply by northword** [reply]
> 快速开始或者可以单列一节，后续的内容可能归到类似“配置/设置”之类的一个大章节？ 给用户一种“走完这节就可以开始用”的积极反馈。然后通过一些例如：要从知网下文献，请按照xxx（链接）操作 的方式将后面的内容挂到快速开始的末尾。 现在看起来快速开始和其他内容并列，里面还层层叠叠的，我看了都有点害怕xs

😂😂😂我觉得可行

**Reply by northword** [reply]
> BTW，刚才发现一个比较严重的问题，飞书文档的搜索功能只有登录后才有，之前一直没发现（因为一直登录着）。我在想开始正式工作之前，讨论下是不是应该换到语雀？我看语雀的官方文档是可以不登录搜索的。 我去试下注册一个语雀账号，能不能不登录搜索。

试了下，语雀和 gitbook 都是可以不登录搜索的，语雀的搜索结果会在新页面显示，gitbook 是当前页面悬浮窗。

**Reply by windingwind** [reply]
@northword 关于之前提到的语雀自定义首页，应该是这个https://www.yuque.com/yuque/thyzgp/zm5k3r。试了下好像可以。
另外文档内容也可以不登录搜索，这一点让我觉得应该考虑下在正式开始之前是不是要换平台

**Reply by windingwind** [reply]
gitbook主要是国内估计访问毫无疑问会有问题，目前想尽快开始整理（这周末我没电脑，只能ipad上做点工作），花太多时间回到选平台的阶段有点太复杂了。

**Reply by northword** [reply]
> @northword 关于之前提到的语雀自定义首页，应该是这个 [https://www.yuque.com/yuque/thyzgp/zm5k3r](https://www.yuque.com/yuque/thyzgp/zm5k3r) 试了下好像可以。 另外文档内容也可以不登录搜索，这一点让我觉得应该考虑下在正式开始之前是不是要换平台

这个我有看过，这个是知识库的首页，并不是团队的首页，不过也可以用上。

官方那样的，在团队的首页列出所有的知识库，
![image](https://user-images.githubusercontent.com/44738481/187012979-463e4a8c-eb3d-423b-993f-febd15772368.png)

并且每个知识库可以返回到团队首页，

![image](https://user-images.githubusercontent.com/44738481/187012965-fb003398-b15b-4895-8781-dcee3273954a.png)

这种似乎我们实现不了，只有语雀官方有。

**Reply by northword** [reply]
之前测试时候建过一个组织，二级域名用了 zotero，zotero.yuque.cn。如果用这个的话就直接下面链接加入就好，如果要换个二级域名的话就新建一个吧。
> https://www.yuque.com/r/organizations/zotero/join?token=R1iOVWKpBWpgOms4# 邀你加入语雀空间「Zotero 中文社区」，体验全新的体系化知识库管理，让协作更高效，让知识成为财富

另外，需要注意语雀免费版只有 10GB 上传流量，（虽然我觉得我们也用不完233）

**Reply by windingwind** [reply]
已加入，但是好像确实没法创建公开的知识库，只能团队内可见。
个人账户可以创建公开知识库

**Reply by northword** [reply]
> 已加入，但是好像确实没法创建公开的知识库，只能团队内可见。
> 个人账户可以创建公开知识库

可以创建公开的知识库，在分享那里，勾选公开到互联网就好。
只是没有公开的团队首页。

**Reply by windingwind** [reply]
试了下只能分享到团队，在未登录情况下访问链接也提示登录，无法直接访问

**Reply by northword** [reply]
> 试了下只能分享到团队，在未登录情况下访问链接也提示登录，无法直接访问

试一下这个

https://zotero.yuque.com/books/share/93bfca90-00d8-4228-8d3a-305863c70f50?# 《项目文档》

![image](https://user-images.githubusercontent.com/44738481/187014469-1ee753ab-2e25-4531-95a8-4556e8a28e45.png)

**Reply by windingwind** [reply]
确实可以。可能因为我不是管理员所以没有这个选项，这样看语雀也许比飞书更方便（可以未登录搜索）？

**Reply by northword** [reply]
> 这样看语雀也许比飞书更方便（可以未登录搜索）

嗯，就阅读而言是的。

> 可能因为我不是管理员所以没有这个选项

语雀这个权限好多... 空间一个权限，团队一个权限，知识库还有单独的权限，全员团队居然还不是自动加入 — — ，我只设置了你为空间管理员，没设置团队管理员。

ps. 还是 GitBook 好用，要不是网络问题...

**Reply by windingwind** [reply]
确实。
我目前倾向于语雀，反正现在还没开始正式动工。其他几位怎么看？
@zotero-cn/all

**Reply by windingwind** [reply]
如果换到语雀，其他两位也同意。不如我们就从你的语雀团队里面开始正式动工文档？@northword

**Reply by northword** [reply]
> 如果换到语雀，其他两位也同意。不如我们就从你的语雀团队里面开始正式动工文档？@northword

我都可以的。

**Reply by windingwind** [reply]
@northword 已将目录整理到语雀。稍微调整了部分章节目录以避免太深的目录层级

**Reply by northword** [reply]
> @northword 已将目录整理到语雀。稍微调整了部分章节目录以避免太深的目录层级

棒，这个目录我觉得 ok。

不过得先说声抱歉，我这段时间在改小论文补数据，返修时间比较紧，所以可能这几天在文档上的精力会比较少，我尽量参与起来。

**Reply by windingwind** [reply]
没关系，我也有空会一起整理。
在语雀空间-团队-知识库的三级结构中，好像只有团队（现在是全员团队）的路径无法自定义：https://zotero.yuque.com/staff-gkhviy/zotero
中间有一个staff-xxx的东西。不知道创建者能够自定义吗？
![image](https://user-images.githubusercontent.com/33902321/188080917-62d8987c-d72a-41a1-ae64-34b830a21c8c.png)
我试着新建了一个团队，路径看起来短一点，但还是蛮奇怪的。要是能自定义就好了

**Reply by northword** [reply]
我也不能自定义，（咱俩能干的事情应该是一样的），
这个路径似乎没什么影响，因为分享链接不是 https://zotero.yuque.com/staff-gkhviy/zotero 这种的，而是 https://zotero.yuque.com/books/share/93bfca90-00d8-4228-8d3a-305863c70f50?# 《Zotero 百科全书》 ，后面是uuid，不包括团队的路径

我找到了语雀官方那种首页，包括开源阅读这种 https://www.yuque.com/legado  首页，似乎使“知识小组”功能，

![image](https://user-images.githubusercontent.com/44738481/188087716-a20a35ea-63c3-4248-89b8-e4e3cb3e5489.png)

不过这个功能入口下线了...  （https://www.yuque.com/yuque/qa/neyorv）

**Reply by windingwind** [reply]
那倒是不要紧，就这样先弄起来再说了

**Reply by windingwind** [reply]
已完成快速上手章节。关于引用于CSL方面，我不是特别熟悉，还要麻烦其他朋友帮忙撰写完善。我接下来主要将整理搬运常见问题方面的文档。

**Reply by zepinglee** [reply]
> 已完成快速上手章节。关于引用于CSL方面，我不是特别熟悉，还要麻烦其他朋友帮忙撰写完善。我接下来主要将整理搬运常见问题方面的文档。

我去看一下。

**Reply by windingwind** [reply]
已完成同步章节与相关常见问题。

**Reply by windingwind** [reply]
> > 已完成快速上手章节。关于引用于CSL方面，我不是特别熟悉，还要麻烦其他朋友帮忙撰写完善。我接下来主要将整理搬运常见问题方面的文档。
> 
> 我去看一下。

主要是与CSL相关的一些内容，可能可以放进常见问题章节（例如，最典型的怎样利用CSL仓库查找自己需要的样式，怎样编辑，等等）

**Comment by windingwind** [comment]
邀请你加入Zotero Plugins，可通过 8 位企业邀请码 XLGBVMWB 加入企业。查看如何使用企业邀请码：https://www.feishu.cn/hc/articles/360040931334
你也可通过点击此链接 https://w6zjinn49j.feishu.cn/invite/member/A2Z7_xm8mmNtXH4a 快速加入企业

**Reply by windingwind** [reply]
@northword

**Reply by northword** [reply]
收到
